Dockerfile

```diff
@@ -93,8 +93,10 @@ RUN cd /home/fwdump/coreboot/util/nvramtool && \
 RUN cd /home/fwdump/coreboot/util/intelp2m && \
 	make && cp intelp2m /usr/local/sbin/
 
+# flashrom with Tiger Lake support
 RUN cd /home/fwdump/flashrom && \
-	git checkout v1.2 && \
+	git fetch https://review.coreboot.org/flashrom refs/changes/78/55578/13 && \
```

Comment:
No longer needed now that Tiger Lake support has been merged

Reply:
Feel free to fix :)

Reply:
Fixed